### DISCUSSION TITLE
HS Utilities Extension

### DISCUSSION THREAD

**Discussion by ghost** [TurboWarp/extensions#643] [Show and tell]
I made my own unsandboxed extension called HS Utilities.
I don't have a preview.

The blocks are:
<is () an integer?>
<either <> or <>>
forever hat block

It's the simplest possible unsandboxed extension.

Update: Added 3 new blocks.
(random chinese character)
when <>is true
when <> is false

I hope you enjoy my custom extension, and hopefully it gets on the gallery one day.

**Comment by snowboyz0825** [comment]
where is it?
